### ISSUE TITLE
Change password default

### ISSUE THREAD

**Issue by Mte90** [lkwdwrd/generator-vvv#43]
Is impossible set the user and password. Also they are different form the original vvv.
Now are `wordpress` and `wordpress` while on vvv are `admin` and `password`.

**Comment by robert-isaev** [comment]
+1

**Comment by AaronHolbrook** [comment]
Agreed, I prefer to use dev/dev for all my installs - would be nice to let the user override define these defaults

**Comment by lkwdwrd** [comment]
This is available in the new version. Most of the current issues are actually resolved now, but I've been waiting until it's stable enough to hit NPM as version 1.0.0

For WP password customization see the [manifest schema documentation for WP options](https://github.com/lkwdwrd/generator-vvv/blob/version1/docs/manifest-schema.md#admin-user)
